src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -38,7 +38,7 @@ public static partial class PlatformDetection
         public static bool IsNotArm64Process => !IsArm64Process;
         public static bool IsArmOrArm64Process => IsArmProcess || IsArm64Process;
         public static bool IsNotArmNorArm64Process => !IsArmOrArm64Process;
-        public static bool IsArgIteratorSupported => IsWindows && IsNotArmNorArm64Process;
+        public static bool IsArgIteratorSupported => IsWindows && IsNotArmProcess;
```

Comment:
Should this be just `IsWindows` to match https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/ArgIterator.cs#L30 ?

Reply:
I thought so too but I figured there must be some other incompatibility hence the condition (albeit unfortunately not throwing PNSE). 

@jashook does this feature work on 32 bit ARM?

Reply:
This should not be expected to work on windows arm32